src/main/java/frc/robot/subsystems/base/common/TimerBase.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.base.common;
+
+/**
+ * Add methods that indicated that the subsystem needs to use timer.
+ * <p>
+ *
+ * @author Barel
+ * @version 1.0
+ * @since 2020-off
+ */
+public interface TimerBase {
```

Comment:
```suggestion
public interface TimerBased {
```
A subsystem is _based on_ a timer, it is not _a_ timer.